xbmc/platform/freebsd/CPUInfoFreebsd.cpp

```diff
@@ -0,0 +1,149 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "CPUInfoFreebsd.h"
+
+#include "utils/log.h"
+#include "utils/Temperature.h"
```

Comment:
wrong include order

Reply:
how?

Reply:
@Rechi it's really hard to push this PR through if you don't answer my questions

Reply:
> Included header files have to be sorted (case sensitive) alphabetically [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
still wrong